packages/botonic-plugin-contentful/src/cms/contents.ts

```diff
@@ -172,7 +172,8 @@ export class Queue extends Content implements ContentWithKeywords {
     readonly queue: string,
     readonly shortText?: string,
     readonly schedule?: Schedule,
-    readonly searchableBy: SearchableBy = new SearchableBy()
+    readonly searchableBy: SearchableBy = new SearchableBy(),
+    readonly indicationText?: string
```

Comment:
please confirm with somebody from enterprise (Candice) or with a native speaker (Gordon?) that this is the best name for this concept. 
my brainstorm:  handoffMessage